### PR TITLE
Add delay after each command to slow videos down

### PR DESCRIPTION
**Pull request by NeilLuptonMoJ**
Creating videos from the Cypress tests results in very high speed output,
this adds delays to each step to make the output eaiser to view.

## What does this pull request do?

Adds a delay, configured by an env setting to each command

## What is the intent behind these changes?

Should allow us to set the cypress tests up as user journeys, which can then be outputted as videos so that they can be used as demos.

### CODE REVIEW

File: integration_tests/support/delay.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-restricted-syntax */
+
+// Set COMMAND_DELAY above zero for demoing to stakeholders,
+const COMMAND_DELAY = Cypress.env('COMMAND_DELAY') || 0
```
Comment (sldblog) [inline integration_tests/support/delay.js:4]: Genuine question, not a veiled suggestion: should this be namespaced to `CYPRESS_COMMAND_DELAY`?

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "test": "jest --runInBand",
     "int-test": "cypress run --config video=false",
     "int-test-ui": "cypress open",
+    "int-test-video": "CYPRESS_NO_COMMAND_LOG=1 cypress run --config video=true,slowTestThreshold=15000 --env COMMAND_DELAY=1200",
```
Comment (sldblog) [inline package.json:28]: Do we need the `slowTestThreshold`? Seems dissimilar to the rest of the changes

**Review by sldblog — APPROVED** [review]
LGTM 👍 No blockers

File: package.json (hunk 2 above)
Comment (sldblog) [reply package.json:28]: Is this is to avoid the delay triggering slow test warnings?

Comment (NeilLuptonMoJ) [reply package.json:28]: yeah, adding the delays causes all the tests to trigger the slow warnings, so resetting the threshold for these tests

File: integration_tests/support/delay.js (hunk 1 above)
Comment (NeilLuptonMoJ) [reply integration_tests/support/delay.js:4]: Good point, I'll update this

**Pull request merged**